tools/fix_timepoint_date_problems.php

```diff
@@ -313,13 +313,21 @@ function addInstrument($sessionID, $testName)
      */
     // feedback object
     print $user->getUsername();
-    $feedback =& NDB_BVL_Feedback::singleton($user->getUsername(), null, $sessionID, $commentID);
+    $feedback =& NDB_BVL_Feedback::singleton($user->getUsername(), null, $sessionID);
     if (PEAR::isError($feedback)) {

```

Comment:
Since you're already modifying this, can you remove the 2 PHP 4 style exceptions that show up in this diff?